pkg/agents/team7/agent2/agent.go

```diff
@@ -0,0 +1,61 @@
+package team7agent2
+
+import (
+	"github.com/SOMAS2021/SOMAS2021/pkg/infra"
+	"github.com/SOMAS2021/SOMAS2021/pkg/utils/globalTypes/food"
+)
+
+type CustomAgent1 struct {
+	*infra.Base
+	// new params
+}
```

Comment:
```suggestion
type CustomAgent2 struct {
	*infra.Base
	// new params
}
```
You should change all instances of this.